index.txt

```diff
@@ -913,7 +913,7 @@ as [Hahn2015]_, eq. (15)
     \breve{S}_n^m(\phi_\text{s},r_\text{s},\w) =
         -\pi \i^{m-n+1}
         \Hankel{2}{m}{\wc r_\text{s}}
-        Y_n^{-m}(\frac{\pi}{2},\phi_\text{s}).
+        Y_n^{-m}(0,\phi_\text{s}).
```

Comment:
Yes, it is supposed to be the plane perpendicular to the line source.

In my paper, \theta indicates the angle from the north pole which is different from here.